saleor/graphql/csv/types.py

```diff
@@ -0,0 +1,46 @@
+import graphene
+from graphene import relay
+from graphene_federation import key
+from graphql_jwt.exceptions import PermissionDenied
+
+from ...core.permissions import AccountPermissions
+from ...csv import models
+from ..core.connection import CountableDjangoObjectType
+from .enums import JobStatusEnum
+
+
+@key(fields="id")
```

Comment:
Not sure if this is needed for this type. @salwator I know it's related to Federation, do you think we need to support this for the export job type?

Reply:
I removed that.